### DISCUSSION TITLE
Création d'un site de simulation d'entretien

### DISCUSSION THREAD

**Discussion by 76633106SY** [pallets/flask#5500] [Q&A]
Bonjour à la communauté. Je suis débutant dans le développement web. J'ai un code python pour simuler des entretiens et un code html pour l'interface l'utilisateur. Je n'arrive pas à les lier. Pouvez-vous m'aider?

**Comment by ThiefMaster** [comment]
- Most of us don't speak French. Please use English.
- Even after using Google Translate, it's not clear what you are asking about and how it's related to Flask.

**Comment by philibe** [comment]
@76633106SY

Il faut toujours poster  le code source  exemple reproductible du source que l'on a essayé. Personne ne pourra t'aider sans cela.
De plus pour rester dans le sujet il faut être sûr que ça soit du Flask.

En général les discussions dans Github sont des questions la majeure partie du temps sur l'architecture du produit.

En fait je crains que sans ces infos ta question n'ait pas sa place ici mais davantage sur Stackoverflow, mais  toujours avec un code source  exemple reproductible du source que tu as essayé sinon tu vas te faire jeter :) 

Et en anglais impérativement que ça soit ici ou sur StackOverflow !

**Reply by 76633106SY** [reply]
Thank you for advice

**Comment by 76633106SY** [comment]


**Reply by ThiefMaster** [reply]
You need to revoke that API token you just posted right now.

**Reply by 76633106SY** [reply]
I do it now. Thank you for your advice
